internal/security/proxy/config/config.go

```diff
@@ -76,6 +76,7 @@ type KongAclInfo struct {
 }
 
 type SecretServiceInfo struct {
+	Scheme          string
```

Comment:
This should be Protocol to be consistent with every other service

Reply:
`Protocol` is an attribute of `SecretStore`, not `SecretService` and I pointed out this and other inconsistencies in security configuration in issue #2495, which was punted until we could make breaking changes...

Reply:
Should this issue #2495 be non-issue any more if we go to `non-TLS` approach because there is no CA cert to deal with in the code in Ireland release?

Reply:
`Protocol` is the standard name we use. Using `Scheme` creates more inconsistencies.